doc/api/buffer.md

```diff
@@ -71,6 +71,18 @@ first argument, security and reliability issues can be inadvertently introduced
 into applications when argument validation or `Buffer` initialization is not
 performed.
 
+For example, if an application can be convinced to call `new Buffer(100)`
```

Comment:
```suggestion
For example, if an application calls `new Buffer(100)`
```

Reply:
the convincing to do this instead of the other is what is important here, I think. Maybe some other wording would make that more clear?

Reply:
Maybe this?

```suggestion
For example, if an attacker can cause an application to call `new Buffer(100)`
```

...or maybe this?

```suggestion
For example, if a bug makes it possible for an application to call `new Buffer(100)`
```

Reply:
How about "if an attacker can cause a number to be passed to an application that assumes it will always be passed strings, it may call `new .."?

Reply:
That has the virtue of clarity. 👍 That's good enough for me. I'd use somewhat different wording, but either yours above or mine below is fine by me:

If an attacker can cause an application to receive a number where a string is expected, the application may call `new....

Reply:
used yours